data/protections-disease-gene.tsv

```diff
@@ -0,0 +1,8 @@
+omim_id	mondo_id	mondo_label	orcid	comment
```

Comment:
Please re-named from protections to protected.

Reply:
OK I'm pushing a new commit for that. But if we do that, let me know if you want me to change "exclusions" to "excluded".